ceph.spec.in

```diff
@@ -434,6 +434,7 @@ Requires:       python%{_python_buildid}-six
 %if 0%{?fedora} || 0%{?rhel}
 Requires:       python%{_python_buildid}-cherrypy
 Requires:       python%{_python_buildid}-werkzeug
+Requires:       libstoragemgmt
```

Comment:
if this is not a feature only works for rpm distros, why don't we also add this in `debian/control` ?

also, could you enlighten me how ceph-mgr or its module(s) pulls in libstoragemgmt? i grep'ed `storagemgmt`, but nothing shows up.

Reply:
the rook module will use the ceph/ceph image to run lsmcli to turn on blinking lights.  the ssh module will presumably do something similar in the future.  including it in the ceph image is expedient... easier than having a separate image just for the purpose.

Reply:
would it make sense to add these dependencies to the `mgr-module-ssh` and `mgr-module-rook` packages? `mgr` will not need libstoragemgmt per se.

Reply:
@tchaikov I haven't added it to the Debian portion because I haven't looked if Debian has the correct version for it. Let me do a quick search for this...

@sebastian-philipp `mgr` does not require it but having it a single place is better than each time adding it to every single module that needs it. Otherwise, we need a new package like `ceph-mgr-orchestrator-common`. This makes me think that we should have named orchestrator packages differently, for instance `ceph-mgr-orchestrator-ssh`.

Reply:
@tchaikov Actually since our official container image is based on CentOS we don't really need it anywhere else. I don't know the state of that package for other distros. If you can ping other distro maintainers that'd be great, otherwise let's move forward with and they will followup later. We really need this for Nautilus :). Thanks.

Reply:
I'm not sure about this. Nothing in ceph actually directly uses libstoragemgmt so I kinda don't see the point of adding it as a dependency of mgr.

Reply:
Fine, I made the change only the orchestrator packages.